zuul.d/edpm_multinode.yaml

```diff
@@ -112,6 +112,8 @@
       cifmw_extras:
         - '@scenarios/centos-9/ci.yml'
         - '@scenarios/centos-9/multinode-ci.yml'
+      cifmw_install_yamls_vars:
+        GALERA_REPLICAS: 3
```

Comment:
This will override the setting in ci.yml:
````
cifmw_install_yamls_vars:
  BMO_SETUP: false
```

Reply:
... we probably want to introduce the hash policy thingy allowing ansible to merge instead of replacing....

Reply:
scenarios/centos-9/ci.yml is used in old e2e job only and new crc edpm job. Can we move `BMO_SETUP: false` and `GALERA_REPLICAS: 3` under cifmw_install_yamls_vars var in ci.yml? this way we donot need to define in the job definition.